### PR TITLE
"fix decode bug"

### PR DESCRIPTION
**Pull request by dzhwinter**
fix https://github.com/PaddlePaddle/Paddle/issues/7683

### CODE REVIEW

File: paddle/framework/lod_tensor_test.cu [hunk 1]
```diff
@@ -1,28 +1,16 @@
-//  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
+//   Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```
Comment (luotao1) [inline paddle/framework/lod_tensor_test.cu]: 我看https://github.com/PaddlePaddle/Paddle/blob/develop/paddle/operators/accuracy_op.cu#L13 是以/* */来进行注释的。

File: benchmark/paddle/image/alexnet.py [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
-#distributed under the License is distributed on an "AS IS" BASIS,
-#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-#See the License for the specific language governing permissions and
-#limitations under the License.
-#!/usr/bin/env python
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```
Comment (luotao1) [inline benchmark/paddle/image/alexnet.py]: 除了这个py的#后面没有加空格，像https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/v2/fluid/backward.py 也没有加。是不是要grep一遍统一修改下？

Comment (dzhwinter) [reply benchmark/paddle/image/alexnet.py]: Done

File: paddle/framework/lod_tensor_test.cu (hunk 1 above)
Comment (dzhwinter) [reply paddle/framework/lod_tensor_test.cu]: go, protobuf, c++ 都支持 "//", 风格和python也统一。自动加的就按"//"吧

File: benchmark/paddle/image/resnet.py [hunk 3]
```diff
@@ -1,16 +1,3 @@
-#  Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserve.
```
Comment (Yancey1989) [inline benchmark/paddle/image/resnet.py]: Why delete the copyright under directory `benchmark/paddle`

Comment (dzhwinter) [reply benchmark/paddle/image/resnet.py]: will be added back to next PR.

**Pull request merged**